i18n.js

```diff
@@ -22,6 +22,7 @@ var vsprintf = require('sprintf-js').vsprintf,
     require('make-plural/data/plurals.json')
   ),
   parseInterval = require('math-interval-parser').default;
+var extend = require('extend');
```

Comment:
should keep the original format of author: using single 'var' declaration statement, one per line